content/user/admin/why_admin.md

```diff
@@ -10,3 +11,7 @@ A somewhat less obvious reason is to permit some configuration actions to be don
 
 Plugin authors sometimes take shortcuts and advise users to modify this file or that one to tune the behavior of a plugin.  Zen Cart deliberately extracted some of those customization choices and put them into an admin panel, so that storeowners can do this by themselves and not need to depend on a developer. 
 
+*This is a tricky balance!*  Many of the same people who complain that the admin is too large and has too many options will also complain that there's no admin switch for the option _they_ want. 
```

Comment:
By "big" and "too large", do people mean "why so many menu options?" or "why so many sections/choices?"

When I first read this I thought "big" and "large" were referring to dimensions or size of the screen or graphics, not to "available options".

Reply:
Yes - they are complaining about the number of options. And complaining the option they want isn't there.  :)